src/components/screens/multiSignature/result/result.js

```diff
@@ -8,34 +8,22 @@ import ProgressBar from '../progressBar';
 import styles from './styles.css';
 
 const Result = ({
-  t, transactionInfo, error, transactions, transactionBroadcasted,
+  t, transaction, error,
 }) => {
-  const [status, setStatus] = useState(!error ? 'pending' : 'fail');
-  const success = status !== 'fail';
-
-  const template = success ? {
+  const template = !error ? {
     illustration: 'registerMultisignatureSuccess',
     message: t('You have successfully signed the transaction. You can download or copy the transaction and share it with members.'),
   } : {
     illustration: 'registerMultisignatureError',
     message: t('Oops, looks like something went wrong.'),
   };
 
-  useEffect(() => {
-    if (transactionInfo) {
-      const confirmed = transactions.confirmed
-        .filter(tx => tx.id === transactionInfo.id);
-      const broadcastError = transactions.broadcastedTransactionsError
-        .filter(tx => tx.transaction.id === transactionInfo.id);
-
-      if (confirmed.length) setStatus('ok');
-      if (broadcastError.length) setStatus('fail');
-    }
-  }, [transactions]);
-
-  useEffect(() => {
-    if (transactionInfo) transactionBroadcasted(transactionInfo);
-  }, [transactionInfo]);
+  const onDownload = () => {
```

Comment:
We already had the same function on another file. maybe lets move to utils and import it from there